tests/test_polydata.py

```diff
@@ -154,13 +154,11 @@ def test_invalid_init():
         pyvista.PolyData(np.array([1]))
 
     with pytest.raises(TypeError):
-        pyvista.PolyData(np.array([1]), 'woa')
+        pyvista.PolyData([1, 2, 3], 'woa')
 
-    with pytest.raises(TypeError):
+    with pytest.raises(ValueError):
         pyvista.PolyData('woa', 'woa')
 
-    with pytest.raises(TypeError):
-        pyvista.PolyData('woa', 'woa', 'woa')
 
```

Comment:
An additional test case to consider:
```py
>>> import pyvista as pv
>>> pv.PolyData([1, 2, 3], [4, 5, 6])
Segmentation fault
```
:smile: We should consider adding some validation to some fundamental utilities. But that can be a different PR.

See also https://github.com/pyvista/pyvista/pull/1150#issuecomment-776968117 and https://github.com/pyvista/pyvista/issues/1097, probably.

Reply:
This would be resolved if we implement a C n_cells method.  For now, it will have to be for another PR.